src/ffi/mod.rs

```diff
@@ -2,6 +2,14 @@
 
 use crate::mem::{CompressError, DecompressError, FlushCompress, FlushDecompress, Status};
 use crate::Compression;
+use std::mem::MaybeUninit;
+
+fn initialize_buffer(output: &mut [MaybeUninit<u8>]) -> &mut [u8] {
+    unsafe {
```

Comment:
It's probably not too common in this codebase, but should we start to explain why this is safe?
The handling of MaybeUninit definitely gives me trouble, while it should be noted that I basically don't use `unsafe` so this looks like a different language to me 😅.